tools/downloader/pytorch_requirements.in

```diff
@@ -0,0 +1,4 @@
+numpy
+torch
+torchvision
+onnx
```

Comment:
A few things need to be done here:

* sort the package names;
* rename the file to `requirements-pytorch.in`;
* replace any mentions of specific dependencies in the documentation with a reference to this file;
* use `pip-compile` to build `ci/requirements-conversion.txt` from a combination of this and MO's `requirements.txt`;
* remove `ci/requirements-mo.txt` (I will amend the pipeline to instead use `ci/requirements-conversion.txt` when it's available).